bin/docker-setup

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
maybe `#!/usr/bin/env bash` ?

Reply:
Done! Thanks @rhymes

Reply:
It's a bit uncommon to use `env` for this. `/bin/sh` is part of POSIX and will execute the system shell, which is either bash, dash or zsh. 

Here's the macOS manpage, but the same is true on other POSIX systems:

```
     sh is a POSIX-compliant command interpreter (shell).  It is implemented by re-execing as either bash(1), dash(1), or zsh(1) as determined by the symbolic
     link located at /private/var/select/sh.  If /private/var/select/sh does not exist or does not point to a valid shell, sh will use one of the supported
     shells.
```

So unless we use any Bash-specific features (we don't), `/bin/sh` is a more natural choice (though I wouldn't necessarily change it again).

Reply:
@citizen428 I agree with you that `sh` is the more common approach, but I think I'll keep the `env` for portability reasons as @rhymes suggested. Thanks.